File: Data Structures & Algorithms/is-anagram/submission-0.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        map<char,int>a;
        map<char,int>b;

        for(auto x:s){
            a[x]++;
        }
        for(auto x:t){
            b[x]++;
        }

        if(a==b){
            return true;
        }
        return false;
    }
};

File: Data Structures & Algorithms/can-place-flowers/submission-0.py
class Solution:
    def canPlaceFlowers(self, flowerbed: List[int], n: int) -> bool:
        size = len(flowerbed)
        i = 0

        while i < size and n > 0:
            left_empty = (i == 0) or (flowerbed[i - 1] == 0)
            right_empty = (i == size - 1) or (flowerbed[i + 1] == 0)

            if flowerbed[i] == 0 and left_empty and right_empty:
                flowerbed[i] = 1 
                n -= 1
            i += 1

        return n <= 0